src/components/AttachmentViewer/AttachmentViewer.tsx

```diff
@@ -29,15 +29,15 @@ import {
   Wrapper,
 } from './styles'
 
-export interface Attachment {
+export interface FormattedAttachment {
   location: string
-  createdAt?: string
-  createdBy?: string
+  createdAt: string | null
```

Comment:
Just for my curiosity. Yesterday you gave feedback that you preferred ?: string to string | undefined since that was shorter and our coding style. Now we change from ? string to string | null. I get that if null is possible it is preferred to undefined, but doesn't the ? indicate the value could be there, it could also not be there?

Reply:
Not sure what you mean with ", but doesn't the ? indicate the value could be there, it could also not be there?" But if you mean that a question mark can represent `null` then that is incorrect. And this is why we use null: 

<img width="741" alt="Screenshot 2023-09-19 at 08 40 07" src="https://github.com/Amsterdam/signals-frontend/assets/46756331/3b76a53a-d306-422f-b3e4-0ca7d2d8141b">